src/components/cookie-consent-drawer.js

```diff
@@ -0,0 +1,29 @@
+import { Button, Drawer } from 'antd';
+import React, { useState } from 'react';
+import cookie from '../common/cookie.js';
+import { useTranslation } from 'react-i18next';
+
+const EDUCANDU_COOKIE_CONSENT = 'educanduCookieConsent';
+
+export default function CookieConsentDrawer() {
+  const consentCookie = cookie.get(EDUCANDU_COOKIE_CONSENT);
+  const [isVisible, setIsVisible] = useState(!consentCookie);
+
+  const handleClose = () => {
+    cookie.set(EDUCANDU_COOKIE_CONSENT, 'true', { expires: 365 });
+    setIsVisible(false);
+  };
+
+  const { t } = useTranslation('cookieConsentDrawer');
+
+  return (
+    <Drawer title={t('consentTitle')} placement="bottom" visible={isVisible} closable={false}>
```

Comment:
I have some trouble with this drawer, it doesn't work like in the documentation, I'm curious if it work better on your machines, please let me know when you have a couple of minutes.

Reply:
Looks good to me (although you didn't really specify what was not working on your machine) :thinking:

Reply:
That's because several things didn't work

Reply:
Solved, thanks!